config/tcs_config.toml

```diff
@@ -14,6 +14,7 @@
 
 [KvStorage]
 StoragePath = "config/Kv_Shared_tmp"
+database_url = "http://localhost:9090"
```

Comment:
Can we add a database mode direct/remote. Based  on the mode the implementation can use either KvStorage directly or remote KvStorage.

Reply:
done as 484fbbe36d0b4c2d970fdcf7fe7a0a30d1a2d5f9